.codeclimate.yml

```diff
@@ -1,3 +1,9 @@
 engines:
   scalastyle:
     enabled: true
+    config:
+      config: src/main/resources/scalastyle_project.xml
+      include_paths:
```

Comment:
`include_paths` isn't normally provided by the user in this way. Code Climate will compute the `include_paths`, which represent the valid workspace for an engine, based on the `exclude_paths` at the top level of a Code Climate configuration file. The `include_paths` are provided within `/config.json` during an engine run: https://github.com/codeclimate/spec/blob/master/SPEC.md#which-files-to-analyze

Reply:
Good to know. I've implemented as you've suggested.